### PR TITLE
About: Add page to about section that explains the AT support tables

### LINKED ISSUE #2970: Define content for About section that explains how to interpret and use the AT support tables
As part of the improvement to the APG support tables we will include a link to explain the data displayed in the table.

This tasks is to define:
- The location of the link
- The text to be displayed for the link
- The text that will a explain the data displayed on the table.  

Some ideas shared by @mcking65 : 

-  include a link above or below the table to “What does this data mean?” or “How to interpret this data” 
- link should be outside of the iframe, i.e., not part of the content returned by the aria-at query. The link could be to a page in the APG about the specific data surfaced in APG and written to address the APG audience with its specific aims. That page could an APG page

### PR DESCRIPTION
**Pull request by mcking65**
Resolve #2970, 3019.

#### Preview Page

[Preview Assistive Technology Support Tables | APG | WAI | W3C](https://deploy-preview-318--aria-practices.netlify.app/aria/apg/about/at-support-tables/) 

___
[WAI Preview Link](https://deploy-preview-318--aria-practices.netlify.app/ARIA/apg) _(Last built on Wed, 22 May 2024 22:23:55 GMT)._

### CODE REVIEW

**Comment by mcking65** [comment]
@boazsender 

Love the clarity, brevity, and structure of your first draft!! Dividing the content into explanations and recommendations is spot-on. I really like your choice of presenting the usage aspects of the content as 'recommendations'.

Thoughts about next rev ...

I think it would be especially helpful to provide a bit more color around the meaning of 'must' and 'should'. We might want to choose two simple example assertions, one 'must' and one 'should' and contrast them, explaining why the must is a must and the should is a should.

In the recommendations, one thing I think we should say more explicitly is that if you are testing your own implementation of a pattern using an AT that scores less than 100% for must and 100% for should, you should determine whether the AT failures are impacting your assessment of your implementation before making changes to your code. If possible,  you should test with an AT that scores 100% if one is available. these recommendations probably fit under "Don’t Code to the Bugs". 


Not coding to the bugs is the most actionable recommendation. It should probably be first.

I am thinking about how to change "Understand the failures" into something more actionable. What does one do with that understanding? Do we really want people to completely avoid implementations of a pattern that fall below 100% must? Perhaps by understanding a failure, they can determine what specific attribute of an implementation is problematic and design around it while remaining within the pattern's guidelines.

File: content/about/at-support-tables/at-support-tables.html [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    <p>
+      As reports on assistive technology interoperability are made available by the ARIA and Assistive Technologies
+      project, the APG Task Force adds a summary of assistive technology support to the relevant pages showing example
+      implementations of APG patterns. This page explains how to interpret and use the assistive technology support
+      summaries.
+    </p>
+
+    <section id="purpose">
+      <h2>Purpose of AT Support Tables</h2>
+      <p>The purpose of the support tables is to provide APG readers with a legible and actionable summary of the
```
[the remark this answers is not in the document]
Comment (boazsender) [reply content/about/at-support-tables/at-support-tables.html:30]: makes sense, updating

File: content/about/at-support-tables/at-support-tables.html [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+          href="https://github.com/w3c/aria-at/wiki/Glossary#assertion-priority">definitions of Must and Should on the
+          project wiki</a>.</p>
+    </section>
+    <section id="recommendations">
+      <h2>Recommendations</h2>
+
+      <h3>Understand the failures</h3>
+      <p>Where possible, APG readers should avoid implementing patterns that have a score lower than 100% in the must
+        column. If, however, you find yourself implementing a pattern with less than 100% in the Must column, you should
+        take time to understand what is failing. You can learn more about the specific failures of a given pattern by
```
[the remark this answers is not in the document]
Comment (boazsender) [reply content/about/at-support-tables/at-support-tables.html:47]: good catch, updating across the doc to "reader"

**Comment by boazsender** [comment]
@mcking65 Thanks, I'm glad you like the first rev.

one question:

> if you are testing your own implementation of a pattern using an AT that scores less than 100% for must and 100% for should, you should determine whether the AT failures are impacting your assessment of your implementation before making changes to your code

Can you say more about why you think this?

File: content/about/at-support-tables/at-support-tables.html [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+        <p>
+          When possible, test implementations of APG patterns with an assistive technology that provides 100% support for both must-have and should-have behaviors.
+        </p>
+      </section>
+
+      <section>
+        <h3>Design to Mitigate Critical Support Failures</h3>
+        <p>
+          Where feasible, avoid designing experiences that rely on features of APG patterns that have less than 100% support for must-have behaviors.
+          If the must-have support level is less than 100% for the example implementation of a pattern, that does not mean all ways of implementing that pattern will present assistive technology users with critical problems.
```
Comment (a11ydoer) [inline content/about/at-support-tables/at-support-tables.html:97]: @mcking65 Can we change "all ways of implementing that pattern" with "multiple ways of implementing that pattern" which was used in #1 suggestion below? 

"If there are multiple implementation examples of..."

File: content/about/at-support-tables/at-support-tables.html [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+      <section>
+        <h3>Don’t Code to the Bugs</h3>
+        <p>
+          ARIA-AT is working with assistive technology vendors to increase their support levels.
+          This means that assistive technologies that align with ARIA-AT interoperability tests will change over time.
+          Exercise caution when implementing a pattern where support levels are less than 100%.
+          Avoid modifying code to accommodate an assistive technology failure unless you are confident that the modification will:
+        </p>
+        <ul>
+          <li>Not negatively affect the experience when using assistive technologies that provide 100% support.</li>
```
Comment (a11ydoer) [inline content/about/at-support-tables/at-support-tables.html:85]: @mcking65 Would it be clearer if we update "that provide 100% support" as "that provide 100% **must-have** support"?

File: content/about/at-support-tables/at-support-tables.html [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+          <li>Not negatively affect the experience when the assistive technology vendor resolves the failure being accommodated.</li>
```
Comment (a11ydoer) [inline content/about/at-support-tables/at-support-tables.html:86]: @mcking65 did you intend to remove "being accommodated" from the the sentence?

**Comment by a11ydoer** [comment]
@mcking65 I reviewed and added three comments for your consideration. The contents look great, and thanks again for creating this useful content for APG users!

**Review by shirsha — APPROVED** [review]

File: content/about/at-support-tables/at-support-tables.html (hunk 4 above)
Comment (mcking65) [reply content/about/at-support-tables/at-support-tables.html:85]: It does not matter if it is a must or should that is being accomodated. However, your question prompted me to clarify by avoiding the need to use support levels as the factor that needs to be considered when making this judgment.

File: content/about/at-support-tables/at-support-tables.html (hunk 5 above)
Comment (mcking65) [reply content/about/at-support-tables/at-support-tables.html:86]: I reworded so it is simpler with a new commit I will push soon.

File: content/about/at-support-tables/at-support-tables.html (hunk 3 above)
Comment (mcking65) [reply content/about/at-support-tables/at-support-tables.html:97]: that would change the meaning. I revised the wording a bit to hopefully resolve the lack of clarity ...

>If the must-have support level is less than 100% for one example implementation of a pattern, that does not mean every other way of implementing that pattern will present assistive technology users with the same problems.

**Comment by mcking65** [comment]
@howard-e Check examples/index.html / coverage (pull_request) is failing.

I looked at the logs and couldn't figure out why. It seems like a diff command is erroring out.

I am going to go ahead and merge. I am guessing the worst case here is that the coverage and quality report is not getting updated. I hope this is something you can easily resolve before we publish. If not, perhaps we publish again with any necessary fix/update.

**Comment by howard-e** [comment]
> @howard-e Check examples/index.html / coverage (pull_request) is failing.
> 
> I looked at the logs and couldn't figure out why. It seems like a diff command is erroring out.

Seems this happened because changes went into `main` from #2775 which affect the coverage report without the script being re-ran (which is also curious) but my guess is the script was just accidentally skipped during a later push on that PR.

> I am going to go ahead and merge. I am guessing the worst case here is that the coverage and quality report is not getting updated. I hope this is something you can easily resolve before we publish. If not, perhaps we publish again with any necessary fix/update.

@mcking65 it was easy enough to resolve, in re-running `npm run coverage-report`. But that made me notice that EVERY example was reported as using `event.keyCode`, the amount reported as using `event.which` which seemed very wrong. That's because the `skipto.js` utility wasn't being ignored during the generation and adding to the metrics. So I've filed #3026 to ignore `skipto.js` and to also re-run the script so report can be up to date, and accurate.

**Pull request merged**